src/main/scala/com/fulcrumgenomics/bam/EstimateRnaSeqInsertSize.scala

```diff
@@ -0,0 +1,267 @@
+import htsjdk.samtools.filter._
+import htsjdk.samtools.util.{CoordMath, Interval, OverlapDetector}
+
+import scala.collection.JavaConversions.iterableAsScalaIterable
+
+@clp(description =
+  """Computes the insert size for RNA-Seq experiments.
+    |
+    |Computes the insert size by counting the # of bases sequenced in transcript space.  The insert size is defined
+    |as the distance between the first bases sequenced of each pair respectively (5' sequencing ends).
+    |
+    |This tool skips reads that overlap multiple genes, reads that aren't fully enclosed in a gene, and reads where the
+    |insert size would disagree across transcripts from the same gene.  Also skips reads that are unpaired, failed QC,
+    |secondary, supplementary, pairs without both ends mapped, duplicates, and pairs whose reads map to different
+    |chromosomes. Finally, skips transcripts where the too few mapped read bases overlap exonic sequence.
+    |
+    |This tool requires each mapped pair to have the mate cigar (MC) tag.  Use SetMateInformation to add the mate cigar.
+    |
+    |The output file will have the extension ".rna_seq_insert_size.txt"
+  """,
+  group = ClpGroups.SamOrBam)
+class EstimateRnaSeqInsertSize
+(@arg(flag="i", doc="Input BAM file.") val input: PathToBam,
+ @arg(flag="r", doc="Input gene annotations in refFlat form.  Format described here: http://genome.ucsc.edu/goldenPath/gbdDescriptionsOld.html#RefFlat")
+ val refFlat: FilePath,
+ @arg(flag="p", doc="Output prefix file to which to write.") val prefix: Option[PathPrefix] = None,
+ @arg(flag="d", doc="Include duplicates") val includeDuplicates: Boolean = true,
+ @arg(flag="D", doc= "Generate mean and standard deviation by filtering to MEDIAN + DEVIATIONS*MEDIAN_ABSOLUTE_DEVIATION. "
+   + "This is done because insert size data typically includes enough anomalous values from chimeras "
+   + "and other artifacts to make the mean and sd grossly misleading regarding the real distribution."
+ ) val deviations: Double = 10.0,
+ @arg(flag="q", doc="Ignore reads with mapping quality less than this value.") val minimumMappingQuality: Int = 30,
+ @arg(flag="m", doc="The minimum fraction of read bases that must overlap exonic sequence in a transcript") val minimumOverlap: Double = 0.95
+) extends FgBioTool with LazyLogging {
+  import scala.collection.JavaConversions.{asScalaIterator, seqAsJavaList}
+
+  Io.assertReadable(input)
+  Io.assertReadable(refFlat)
+  prefix.foreach(Io.assertCanWriteFile(_))
+
+  override def execute(): Unit = {
+    val progress            = new ProgressLogger(logger, verb = "read", unit = 5e6.toInt)
+    val in                  = SamReaderFactory.make.open(input)
+    val refFlatSource       = RefFlatSource(refFlat, Some(in.getFileHeader.getSequenceDictionary))
+    val geneOverlapDetector = new OverlapDetector[Gene](0, 0)
+    val counters            = PairOrientation.values().toList.map { pairOrientation => (pairOrientation, new NumericCounter[Long]()) }.toMap
```

Comment:
`map` was showing up before, weird, perhaps I imported a scala conversion later and now it works?